en-US/Language.xml

```diff
@@ -986,6 +1030,56 @@ Numbers (move from N entry) -->
 				</listitem>
 
 			</varlistentry>
+<!-- Added entry for "easy, simple" -->
+			<varlistentry id="easy">
+				<term> Avoiding "Easy" </term>
+				 <listitem>
+					<para>
+						Avoid stating that a task is easy to do or that a product is easy to use. Avoid "friendly" and "user-friendly" and similar terms. Focus instead on how to accomplish a task or on relevant features of a product.
+					</para>
+					 <table>
+						<title></title>
+						 <tgroup cols="2" colsep="1" rowsep="1">
+							<colspec colname="c1"></colspec>
+							 <colspec colname="c2"></colspec>
+							 <thead>
+								<row>
+									<entry> Example </entry>
+									 <entry> Improvement </entry>
+
+								</row>
+
+							</thead>
+							 <tbody>
+								<row>
+									<entry> Mounting a device is relatively simple. 
+									Identify the device for mounting, ensure that the mount point exists, and mount the device on the mount point.</entry>
+									 <entry> (Omit the first sentence.) 
+									 To mount a device, identify the device for mounting, ensure that the mount point exists, and mount the device on the mount point. </entry>
+
+								</row>
+								<row>
+									<entry> Kubernetes is an orchestration service that makes it easier to deploy, manage, and scale container-based applications across a cluster of container hosts. </entry>
+									 <entry> 
+									<para>
+									Option 1: Kubernetes is an orchestration service that deploys, manages, and scales container-based applications across a cluster of container hosts. 
+									</para>
+									<para>
+									Option 2: Kubernetes is an orchestration service that makes it easier than manual container management to deploy, manage, and scale container-based applications across a cluster of container hosts.
```

Comment:
Please talk to Fernando about this one.

Reply:
Updated following feedback from Fernando.